migration-engine/migration-engine-tests/tests/native_types/mysql.rs

```diff
@@ -740,7 +740,7 @@ fn filter_to_types(api: &TestApi, to_types: &'static [&'static str]) -> Cow<'sta
     to_types.into()
 }
 
-#[test_connector(tags(Mysql))]
+#[test_connector(tags(Mysql), exclude(TiDB))]
```

Comment:
Does this not work in TiDB? Is there an issue?

Reply:
@Mini256  this one as well :)

Reply:
I deleted the `exclude` on this test case and filtered out some unsupported type conversions in TiDB to make this test run: https://github.com/prisma/prisma-engines/pull/3308/commits/949008f5bcf1bd4a618901303547c4a1b546a67d

I found that TiDB does not support the following type conversions, but I need to further confirm with our R&D team next week.

<table>
<tr>
    <th>from_type</th>
    <th>to_type</th>
    <th>reason</th>
<tr>
<tr>
    <td>Binary(8)</td><td>Char(64)</td><td rowspan="6">Unsupported modify charset from utf8mb4 to binary</td>
</tr>
<tr>
    <td>Binary(8)</td><td>VarChar(20)</td>
</tr>
<tr>
    <td>Binary(8)</td><td>LongText</td>
</tr>
<tr>
    <td>Binary(8)</td><td>MediumText</td>
</tr>
<tr>
    <td>Binary(8)</td><td>TinyText</td>
</tr>
<tr>
    <td>Binary(8)</td><td>Text</td>
</tr>
<tr>
    <td>Char(10)</td><td>Blob</td><td rowspan="4">Unsupported modify charset from utf8mb4 to binary</td>
</tr>
<tr>
    <td>Char(10)</td><td>LongBlob</td>
</tr>
<tr>
    <td>Char(10)</td><td>MediumBlob</td>
</tr>
<tr>
    <td>Char(10)</td><td>TinyBlob</td>
</tr>
<tr>
    <td>Date</td><td>Bit(64)</td><td>Unsupported modify column: change from original type date to bit(64) is currently unsupported yet</td>
</tr>
<tr>
    <td>Year</td><td>Bit(64)</td><td>Unsupported modify column: change from original type year(4) to bit(64) is currently unsupported yet</td>
</tr>
<tr>
    <td>Binary(8)</td><td>Bit(64)</td><td>Unsupported modify column: change from original type binary(8) to bit(64) is currently unsupported yet</td>
</tr>
</table>

Reply:
Related issue: https://github.com/pingcap/tidb/issues/20700

Reply:
**Action** : Waiting on @Mini256